protocols/gossipsub/src/behaviour/tests.rs

```diff
@@ -60,30 +59,30 @@ where
 
 impl<D, F> InjectNodes<D, F>
 where
-    D: DataTransform + Default + Clone + Send + 'static,
+    D: DataTransform + Clone + Default + Send + 'static,
     F: TopicSubscriptionFilter + Clone + Default + Send + 'static,
 {
-    pub fn create_network(self) -> (Gossipsub<D, F>, Vec<PeerId>, Vec<TopicHash>) {
+    pub fn create_network(&self) -> (Gossipsub<D, F>, Vec<PeerId>, Vec<TopicHash>) {
```

Comment:
Is this signature change necessary? What is the point of keeping the `InjectNodes` builder around? Unless I am missing something, consuming a builder is idiomatic.

Reply:
Done :heavy_check_mark: